app/views/articles/show.html.erb

```diff
@@ -201,7 +201,7 @@
                                                article_id: @article.id) %>
         <% if @display_ad %>
           <div class="pb-4 crayons-layout__comments-display-ad">
-            <%= render partial: "shared/display_ad", locals: { display_ad: @display_ad, data_context_type: DisplayAdEvent::CONTEXT_TYPE_ARTICLE } %>
+            <%= render partial: "shared/display_ad", locals: { display_ad: @display_ad, data_context_type: DisplayAdEvent::CONTEXT_TYPE_ARTICLE, displayed_in_home_feed: false} %>
```

Comment:
Since we already pass `@display_ad` into the partial, I feel like we could access the context of whether this is "displayed_in_home_feed" via that instead of having to add it to all the partials, what do you think?

Reply:
Thanks @benhalpern I have removed `displayed_in_home_feed ` and instead used `display_ad.placement_area`.

Let me know if it can be further improved.